### ISSUE TITLE
Github Insights are not working on this Repo

### ISSUE THREAD

**Issue by Nutomic** [dashpay/dash#2329]
The statistics on this repository are not working:

![screenshot_20181003_003026](https://user-images.githubusercontent.com/1044450/46391643-9feedb00-c6a3-11e8-927c-0ad893e0da08.png)

Here's the link. Most of the other sections are also not working.
https://github.com/dashpay/dash/graphs/contributors

It would be good if this could get fixed, so we can get some more information about Dash development.

**Comment by UdjinM6** [comment]
We do not commit or merge to master directly. Blame github for not being flexible enough 😛

**Comment by Nutomic** [comment]
You should be able to change the default branch of the repo, then it will probably work.

**Comment by nmarley** [comment]
We've discussed this internally and decided to keep `master` as the default branch, as we want the default branch to always be at a stable state.

GitHub's reporting / statistics really aren't super flexible and it would be nice if this could be adjusted per-branch, instead of only on the default one.

**Comment by UdjinM6** [comment]
Probably not, we do not push commits to `develop` directly either. Not sure why github only count direct commits, that's pretty stupid IMO.

**Comment by UdjinM6** [comment]
See https://github.com/bitcoin/bitcoin/graphs/contributors as a proof - it's empty too despite the fact that they merge everything to `master` first and then backport to a specific release branch.

**Comment by Nutomic** [comment]
That is strange. I noticed this because Dash has very low commit/developer numbers on [CryptoCodeWatch](https://cryptocodewatch.com/all), and they claim to use these statistics.

**Comment by nmarley** [comment]
I actually emailed GitHub support about 6 weeks ago for this exact issue, and that's all that is available. And yeah, we squash merge to `develop` so this wouldn't show all the actual commit activity regardless.

TL;DR : GitHub statistics is broken / inflexible.

**Comment by jhodges10** [comment]
Those stats are so bogus @Nutomic. Easiest way to manipulate them and get good numbers is to just commit your package-lock.json from npm and boom you've got tens of thousands of lines of code!

Quality not quantity 👍

**Comment by PastaPastaPasta** [comment]
Discussion resolved. closable @UdjinM6
